src/System.Xml.XDocument/tests/events/EventsAdd.cs

```diff
@@ -264,18 +219,18 @@ public void WorkOnTextNodes1()
                         using (EventsHelper eHelper = new EventsHelper(elem))
                         {
                             n.AddAfterSelf("text0");
-                            TestLog.Compare(elem.Value, "text2text0", "Did not concat text nodes correctly");
+                            Assert.Equal(elem.Value, "text2text0");
```

Comment:
Anywhere we do Assert.Equal, the first argument should be the expected result and the second argument should be the actual one.